rotkehlchen/tests/unit/decoders/test_syncswap.py

```diff
@@ -0,0 +1,336 @@
+from typing import Final
+
+import pytest
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.assets.asset import EvmToken
+from rotkehlchen.chain.evm.constants import ZERO_ADDRESS
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.chain.scroll.modules.syncswap.decoder import (
+    CPT_SYNCSWAP,
+    SYNCSWAP_ROUTER_ADDRESS,
+    SYNCSWAP_VAULT_ADDRESS,
+)
+from rotkehlchen.constants.assets import A_ETH
+from rotkehlchen.constants.misc import ZERO
+from rotkehlchen.constants.resolver import evm_address_to_identifier
+from rotkehlchen.fval import FVal
+from rotkehlchen.history.events.structures.evm_event import EvmEvent
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import ChainID, EvmTokenKind, Location, TimestampMS, deserialize_evm_tx_hash
+
+USDC_SCROLL_EIP155: Final = 'eip155:534352/erc20:0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4'
+USDT_SCROLL_EIP155: Final = 'eip155:534352/erc20:0xf55BEC9cafDbE8730f096Aa55dad6D22d44099Df'
+DAI_SCROLL_EIP155: Final = 'eip155:534352/erc20:0xcA77eB3fEFe3725Dc33bccB54eDEFc3D9f764f97'
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('scroll_accounts', [['0x3D2F6f18c51F3b72382F5Aea85bDe2cFeaB25dc6']])
+def test_scroll_syncswap_eth_usdc_swap(database, scroll_inquirer, scroll_accounts):
+    """Swap ETH for USDC on scroll"""
+    evmhash = deserialize_evm_tx_hash('0xaa2f8589fa581506c067ad128c00dfce35c98daa49d58ea9e27ad4d2bc782084')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=scroll_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    timestamp = TimestampMS(1713530626000)
+    fee = FVal('0.00024694228625118')
+    amount_received_usdc = FVal('36.508481')
+    amount_sent_eth = FVal('0.01180498459310175')
```

Comment:
Here and in the rest of the file. Please keep style consistent.


```suggestion
    timestamp, fee, amount_received_usdc, amount_sent_eth = TimestampMS(.....
```